app/components/MachinesPage/MachinesSuccessMessage.tsx

```diff
@@ -0,0 +1,41 @@
+// LIBRARY IMPORTS
+import {
+  Box,
+  Button,
+  Center,
+  Icon,
+  IconButton,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import { FaRegPaperPlane } from "react-icons/fa6";
+
+// TYPE DEFINITIONS
+type SuccessMessageProps = {
+  handleCloseSuccess: () => void;
+};
+
+// SUCCESS MESSAGE COMPONENT DEFINITION
+export default function MachinesSuccessMessage({
+  handleCloseSuccess,
+}: SuccessMessageProps) {
+  return (
+    <Box position="relative" py={16} h="lg" bg="brand.primary">
```

Comment:
The message looks smaller width-wise when comparing to wireframe; it might be adjusting width based on the content, though I haven't looked into it. 
![Screen Shot 2024-03-06 at 1 04 46 PM](https://github.com/wavecrestweb/kenworthy-machine/assets/109827834/5de27eb0-1dca-4727-9828-7dd584a1ce59)